operator/cmd/mesh/profile-dump.go

```diff
@@ -38,23 +40,31 @@ type profileDumpArgs struct {
 }
 
 const (
-	jsonOutput = "json"
-	yamlOutput = "yaml"
+	jsonOutput  = "json"
+	yamlOutput  = "yaml"
+	flagsOutput = "flags"
 )
 
 const (
-	IstioOperatorTreeString = `
+	istioOperatorTreeString = `
 apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 `
 )
 
+var (
+	// This path value is not stable and should not be part of the output
+	pathBlacklist = map[string]bool{
+		"installPackagePath": true,
```

Comment:
I'd like to understand this better - it could be confusing to users if it's omitted?

Reply:
The field isn't stable.  Before I checked in the code I did `REFRESH_GOLDEN=true go test ./operator/cmd/mesh` and got

```
installPackagePath="/Users/snible/go/src/istio.io/istio/operator/cmd/mesh/testdata/manifest-generate/data-snapshot"
```

This failed gen-check, so I did `BUILD_WITH_CONTAINER=1 time make refresh-goldens` and got

```
installPackagePath="/work/operator/cmd/mesh/testdata/manifest-generate/data-snapshot"
```

This also failed gen-check, which expected something like 

```
installPackagePath="/prow/work/operator/cmd/mesh/testdata/manifest-generate/data-snapshot"
```

I was afraid to tinker with making installPackagePath use relative rather than absolute paths so I decided to blacklist to ensure the value wouldn't change in unit tests.

(If this is not appealing I can modify the unit tests to explicitly remove lines matching `^ installPackagePath=`.)

Reply:
@ostromart Fixed